_data/simple-icons.json

```
@@ -403,6 +403,11 @@
             "hex": "0096D6",
             "source": "https://www.akamai.com"
         },
+        {
+            "title": "Akkoma",
+            "hex": "462D7B",
+            "source": "https://akkoma.dev/assets/img/logo.svg"
```

```suggestion
            "source": "https://akkoma.dev"
```

We can shorten this down to just the homepage, given the SVG is embedded there.